### PR TITLE
fix: match UI for filter Work Items on Work Item Page

### PR DESCRIPTION
**Pull request by SMahil**
fix: match UI with UX for filter Work Items on Work Item Page

#878 

screen shot 2017-02-06 at 11 56 31 am

### CODE REVIEW

File: src/app/work-item/work-item.component.html [hunk 1]
```diff
@@ -29,8 +29,29 @@ <h2 class="card-pf-title text-center">
...
+              <div class="input-group-btn" dropdown>
+                <button class="btn btn-default dropdown-toggle" type="button" id="wi_filter_dropdown" data-toggle="dropdown" dropdown-open>
+                  Filter
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu" aria-labelledby="wi_filter_dropdown">
+                  <li *ngIf="!filters.length">
+                    <span>No filters available</span>
+                  </li>
+                  <li *ngFor="let filter of filters;" role="presentation" (click)="activeFilter(filter.id)">
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item.component.html]: The width of "Assign to me" needs to match the wireframe:
filterscreenshot1
On UI:
filterdropdownui

Comment (SMahil) [inline src/app/work-item/work-item.component.html]: currently what we have is patternfly I used that. 

@fabric8io/uxd Please confirm how it should come

Comment (mindreeper2420) [inline src/app/work-item/work-item.component.html]: @SMahil @nimishamukherjee The Filter Query Dropdown Filter Bar should be utilized:
screen shot 2017-02-07 at 8 33 35 am
http://www.patternfly.org/pattern-library/forms-and-controls/filter/#/_design

Accordingly, it should be as such:
screen shot 2017-02-07 at 8 44 54 am

**Comment by SMahil** [comment]
screen shot 2017-02-08 at 4 33 17 pm

**Comment by mindreeper2420** [comment]
@SMahil it should be 'Assigned to Me' not 'Assign to Me'

**Comment by SMahil** [comment]
screen shot 2017-02-08 at 6 54 25 pm

File: src/app/work-item/work-item.component.html [hunk 2]
```diff
@@ -29,8 +29,42 @@ <h2 class="card-pf-title text-center">
...
+              <div class="input-group-btn" dropdown>
+                <button class="btn btn-default dropdown-toggle" type="button" id="wi_filter_dropdown" data-toggle="dropdown" dropdown-open>
+                  User
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu" aria-labelledby="wi_filter_dropdown">
+                  <li *ngIf="!filters.length">
+                    <span>No filters available</span>
+                  </li>
+                  <li *ngFor="let filter of filters;" role="presentation" (click)="activeFilter(filter.id)">
```
Comment (vikram-raj) [inline src/app/work-item/work-item.component.html]: activeFilter() should not call when the user selects User from the dropdown list.

**Comment by nimishamukherjee** [comment]
@SMahil please rebase.

File: src/app/work-item/work-item.component.ts [hunk 3]
```diff
@@ -60,7 +60,7 @@ export class WorkItemComponent implements OnInit, AfterViewInit {
     if (this.loggedIn) {
       this.filters.push({
         id:  1,
-        name: 'Assign to Me',
+        name: 'Assigned to Me',
```
Comment (kwk) [inline src/app/work-item/work-item.component.ts]: Please make it `Assigned to me`.

**Pull request merged**